<launch>
	<node type="bb2image.py" name="image2bb" pkg="image_inverse_sensor_model" respawn="true" output="screen">
		<param name="topicInName" value="/yolo/BBox/usb_cam" />
		<param name="objectType_obstacle" value="False" />
		<param name="objectType_vehicle" value="False" />
		<param name="objectType_human" value="True" />
	</node>

<!-- 
	<node type="image2ism.py" name="image2ism" pkg="image_inverse_sensor_model" respawn="true" output="screen">
		<param name="objectTypeInt" value="31" />
		<param name="imageWidth" value="800" />
		<param name="imageHeight" value="600" />
		<param name="cam_xTranslation" value="0.0" />
		<param name="cam_yTranslation" value="0.0" />
		<param name="cam_zTranslation" value="1.5" />
		<param name="cam_pitch" value="0.349" />
		<param name="cam_yaw" value="0.1745" />
		<param name="cam_FOV" value="0.349" />
		<param name="grid_resolution" value="0.05" />
		<param name="grid_xSizeInM" value="-1.0" />
		<param name="grid_ySizeInM" value="-1.0" />
	</node>
-->
</launch>
